dotcom-rendering/playwright/lib/load-page.ts

```diff
@@ -72,7 +97,7 @@ const loadPageWithOverrides = async (
 			postData,
 		});
 	});
-	await loadPage(page, path);
+	await loadPage({ page, path, queryParamsOn: false });
```

Comment:
Switch off adding queryparams for tests that POST the data model to DCR as it cause issues.